src/components/sliders/defaults.js

```diff
@@ -73,12 +73,17 @@ function sliderDefaults(sliderIn, sliderOut, layoutOut) {
     coerce('transition.duration');
     coerce('transition.easing');
 
-    Lib.coerceFont(coerce, 'font', layoutOut.font);
-    Lib.coerceFont(coerce, 'currentvalue.font', layoutOut.font);
-
-    coerce('bgcolor', layoutOut.paper_bgcolor);
+    coerce('bgcolor');
```

Comment:
To elaborate slightly, this attribute wasn't used before. Until today it was a hard-coded constant.